src/hip_module.cpp

```diff
@@ -573,7 +620,7 @@ hipError_t hipLaunchCooperativeKernelMultiDevice(hipLaunchParams* launchParamsLi
                 lp.blockDim.x, lp.blockDim.y,
                 lp.blockDim.z, lp.sharedMem,
                 lp.stream, lp.args, nullptr, nullptr, nullptr, 0,
-                true);
+                true, impCoopParams);
```

Comment:
What if this fails?  Should there be some cleanup?

Reply:
added the memory clean up